# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cvs/getline.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur .1/cvs-1.11.23/lib/getline.c .2/cvs-1.11.23/lib/getline.c
--- cvs-1.11.23/lib/getline.c    2009-08-29 23:44:17.202448640 +0200
+++ cvs-1.11.23/lib/getline.c    2009-08-29 23:41:42.000000000 +0200
@@ -155,7 +155,7 @@
 }
 
 int
-getline (lineptr, n, stream)
+my_getline (lineptr, n, stream)
      char **lineptr;
      size_t *n;
      FILE *stream;
diff -ur .1/cvs-1.11.23/lib/getline.h .2/cvs-1.11.23/lib/getline.h
--- cvs-1.11.23/lib/getline.h    2009-08-29 23:44:07.426330066 +0200
+++ cvs-1.11.23/lib/getline.h    2009-08-29 23:38:25.000000000 +0200
@@ -12,7 +12,7 @@
 #define GETLINE_NO_LIMIT -1
 
 int
-  getline __PROTO ((char **_lineptr, size_t *_n, FILE *_stream));
+  my_getline __PROTO ((char **_lineptr, size_t *_n, FILE *_stream));
 int
   getline_safe __PROTO ((char **_lineptr, size_t *_n, FILE *_stream,
                          int limit));


